editor/src/clj/editor/collection.clj

```diff
@@ -525,6 +528,7 @@
         {:keys [id children component-properties]} instance-msg
         is-child? (contains? all-child-ids id)
         build-target-go-props (partial properties/build-target-go-props resource-property-build-targets)
+        component-properties (map source-resource-component-property-desc component-properties)
```

Comment:
The `:clj-value` entries for resources in the properties we get from the build-targets of referenced collections have been converted into build resources. In order for the referenced-resource-to-build-target-mapping to work in `properties/build-target-go-props`, these need to be source resources, not build resources. Here, we map the build resources produced by the call to `properties/build-target-go-props` for the referenced collections build targets back into source resources so we can repeat the process with our own overrides applied.